### ISSUE TITLE
Trouble with DoNotMock and non-default mock frameworks

### ISSUE THREAD

**Issue by andrewparmet** [slackhq/slack-lints#361]
I'm using version 0.8.2 and trying to use the options to support other mocking frameworks in my Android project. I'm finding that my annotation and factories values aren't being respected.

Here's what my `lint.xml` looks like:

```xml
<lint>
    <issue id="DoNotMock">
        <option name="mock-annotations" value="org.mockito.Mock,org.mockito.Spy,io.mockk.impl.annotations.MockK,io.mockk.impl.annotations.SpyK"/>
        <option name="mock-factories" value="org.mockito.Mockito#mock,org.mockito.Mockito#spy,io.mockk.MockKKt#mockk,io.mockk.MockKKt#mockkClass,io.mockk.MockKKt#mockkObject,io.mockk.MockKKt#spyk,com.nhaarman.mockitokotlin2.MockingKt#mock,com.nhaarman.mockitokotlin2.SpyingKt#spy,org.mockito.kotlin.MockingKt#mock,org.mockito.kotlin.SpyingKt#spy"/>
    </issue>
</lint>
```

Lint detects usages of the default Mockito symbols but none of the others. In fact no change I make appears to be respected - Mockito symbols are still detected even if I set the properties to be empty strings.

I do know that this XML file is being picked up by lint - if I set the severity to `ignore` then no issues are reported at all.

Any suggestions for where I might be doing something wrong? Thanks!

**Comment by andrewparmet** [comment]
I've discovered if I set the options on the DoNotMockRecordClass issue they're picked up correctly by DoNotMock. My guess is there's some lazy/only-once loading going on that's interfering.
